### PR TITLE
Add a guide for securing WildFly apps on OpenShift with the Auth0 OpenID provider

### PR DESCRIPTION
**Pull request by fjuma**
https://fjuma.github.io/wildfly-elytron/blog/securing-wildfly-apps-auth0-openshift/

### CODE REVIEW

File: _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc [hunk 1]
```diff
@@ -0,0 +1,303 @@
+---
+layout: post
+title: 'Securing WildFly Apps with Auth0 on OpenShift'
+date: 2023-12-08
```
Comment (skyllarr) [inline _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc]: just a minor to not forget to update the date

File: _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+. Using the sidebar menu on the left side of the Dashboard, navigate to the *APIs* page and copy the *API Audience* value.
+
+. Using the sidebar menu on the left side of the Dashboard, navigate to the *Settings* page and scroll down to the *API Authorization Settings*. Paste the API Audience value you just copied into the *Default Audience* field and then click on *Save*.
+
++
+This will allow us to receive access tokens that are JWTs from Auth0. In the future, we're hoping to add the ability
+to handle opaque access tokens as well to WildFly's Elytron OIDC Client subsystem.
+
+. Using the sidebar menu on the left side of the Dashboard, click on *User Management* and then *Users*. You can then
+create a new user by clicking on *Create User*. You'll need to specify the new user's email, we'll use *user@example.com*. You'll also need to set a password for the user.
```
Comment (skyllarr) [inline _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc]: Just a minor question, I noticed in the video that the user@example.com was already existing, and was just wondering if it is created automatically or it was created before recording of the video? Nonetheless, this looks good to me!

**Review by skyllarr — APPROVED** [review]
LGTM!

File: _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc (hunk 2 above)
Comment (fjuma) [reply _posts/2023-12-08-securing-wildfly-apps-auth0-openshift.adoc]: I created the user@example.com user following the steps here before recording the video (just to make the video a bit shorter).

**Pull request merged**